src/Controllers/ElementalAreaController.php

```diff
@@ -354,25 +353,19 @@ public function save(array $data, Form $form): HTTPResponse
 
     /**
      * Remove the pseudo namespaces that were added to form fields by the form factory
-     *
-     * @param array $data
-     * @param int $elementID
-     * @return array
      */
-    public static function removeNamespacesFromFields(array $data, $elementID)
+    private function removeNamespacesFromFields(Form $form, int $elementID): void
```

Comment:
This method wasn't used anywhere else, so no need for it to be public nor static.